server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+openapi: 3.0.0
+info:
+  description: >
+    _This document describes the REST API of Ark Automate._
+    Ark Automate is a platform that allows office users and software developers to automate business or everyday processes by simply sketching the steps of their process. By using simple flowcharts or powerful BPMN in their process outlines, users can create small software solutions using RPA that finish their tasks much faster and more reliably.
+  version: 1.0.0
+  title: Ark API
+  license:
+    name: LICENSE (MIT)
+    url: https://github.com/bptlab/ark_automate
```

Comment:
Have we ever agreed on a license? I can't remember it. Maybe we should have a look, if we're allowed to use the MIT license and if there are any problems with startup/dependencies we have

Reply:
Oh you are right. I thought we would use MIT but if look at the readme we haven't mentioned any lincense yet. I will delete this part for now and create a user story for that so that we can agree on a license in the last sprint :)